src/Elliptic/Systems/Xcts/FirstOrderSystem.hpp

```diff
@@ -0,0 +1,261 @@
+ * + \alpha\psi \left( \frac{7}{8}\psi^{-8} \bar{A}^2
+ * + \frac{5}{12} \psi^4 K^2 + \frac{1}{8}\bar{R}
+ * + 2\pi\psi^4\left(\rho + 2S\right) \right)
+ * - \psi^5\partial_t K + \psi^5\left(\beta^i\bar{D}_i K
+ * + \beta_\mathrm{background}^i\bar{D}_i K\right) \\
+ * f_{\alpha\psi} &= 0 \\
+ * F^i_{v_j} &= \delta^i_j \psi \\
+ * S_{v_j} &= v_j \\
+ * f_{v_j} &= 0 \\
+ * F^i_{B_{jk}} &= \delta^i_{(j} \beta_{k)} \\
+ * S_{B_{jk}} &= B_{jk} + \bar{\Gamma}_{ijk}\beta^i \\
+ * f_{B_{jk}} &= 0 \\
+ * F^i_{w_j} &= \delta^i_j \alpha\psi \\
+ * S_{w_j} &= w_j \\
+ * f_{w_j} &= 0 \\
+ * \text{with} \quad \bar{A}^{ij} &= \frac{\psi^7}{2\alpha\psi}\left(
+ * \left(\bar{L}\beta\right)^{ij} +
+ * \left(\bar{L}\beta_\mathrm{background}\right)^{ij} - \bar{u}^{ij} \right) \\
+ * \text{and} \quad \left(\bar{L}\beta\right)^{ij} &=
+ * 2\left(\bar{gamma}^{ik}\bar{gamma}^{jl} - \frac{1}{3}
+ * \bar{gamma}^{jk}\bar{gamma}^{kl}\right) B_{kl}
+ * \f}
+ *
+ * Note that the symbol \f$\beta\f$ in the equations above means
+ * \f$\beta_\mathrm{excess}\f$. The full shift is \f$\beta_\mathrm{excess} +
+ * \beta_\mathrm{background}\f$. See `Xcts::Tags::ShiftBackground` and
+ * `Xcts::Tags::ShiftExcess` for details on this split. Also note that the
+ * background shift is degenerate with \f$\bar{u}\f$ so we treat the quantity
+ * \f$\left(\bar{L}\beta_\mathrm{background}\right)^{ij} - \bar{u}^{ij}\f$ as a
+ * single background field (see
+ * `Xcts::Tags::LongitudinalShiftBackgroundMinusDtConformalMetric`). The
+ * covariant divergence of this quantity w.r.t. the conformal metric is also a
+ * background field.
+ *
+ * \par Solving a subset of equations:
+ * This system allows you to select a subset of `Xcts::Equations` so you don't
+ * have to solve for all variables if some are analytically known. Specify the
+ * set of enabled equations as the first template parameter. The set of required
+ * background fields depends on your choice of equations.
+ *
+ * \par Conformal background geometry:
+ * The equations simplify significantly if the conformal metric is flat
+ * ("conformal flatness") and in Cartesian coordinates. In this case you can
+ * specify `Xcts::Geometry::FlatCartesian` as the second template paramter so
+ * computations are optimized for a flat background geometry and you don't have
+ * to supply geometric background fields. Else, specify
+ * `Xcts::Geometry::Curved`.
+ *
+ * \par Conformal matter scale:
+ * The matter source terms in the XCTS equations have the known defect that they
+ * can spoil uniqueness of the solutions. See e.g. \cite Baumgarte2006ug for a
+ * detailed study. To cure this defect one can conformally re-scale the matter
+ * source terms as \f$\bar{\rho}=\psi^n\rho\f$, \f$\bar{S}=\psi^n S\f$ and
+ * \f$\bar{S^i}=\psi^n S^i\f$ and treat the re-scaled fields as
+ * freely-specifyable background data for the XCTS equations. You can select
+ * the `ConformalMatterScale` \f$n\f$ as the third template parameter.
```

Comment:
question, is there a "Right" value of `n` or does that depend on either the physical setup or the matter type?

Reply:
I added a note to the docs. https://arxiv.org/abs/0804.3787 suggests n=6 to make volume integrals independent of the conformal factor (see footnote 1). @haraldp271 would you like to add some insight? :)

Reply:
The big thing is to have n big enough for the correct sign.  Beyond that, I am not aware of a strongly preferred value (n=6 is nice, as the  integrated matter density independent of conformal factor)